salt/utils/vt.py

```diff
@@ -156,6 +160,9 @@ def __init__(self,
         self.child_fd = None
         self.child_fde = None
 
+        self.patrial_data_stdout = b''
```

Comment:
should this be `partial`

Reply:
Yes, yes it should. Thanks for catching that typo.